### PR TITLE
removed namspace fields and enable check for mysql

### PR DESCRIPTION
**Pull request by logoutdhaval**

### CODE REVIEW

File: helm/fineract/values.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@ selfservice:
 
 
 mysql:
+  fullnameOverride: fineract-mysql
```
Comment (fynmanoj) [inline helm/fineract/values.yaml:81]: Check the indentation once

File: helm/fineract/Chart.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```
Comment (fynmanoj) [inline helm/fineract/Chart.yaml:1]: Why this version is upgraded?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Clarification requested

File: helm/fineract/values.yaml (hunk 1 above)
Comment (logoutdhaval) [reply helm/fineract/values.yaml:81]: Yes, indentation is correct.

File: helm/fineract/Chart.yaml (hunk 2 above)
Comment (logoutdhaval) [reply helm/fineract/Chart.yaml:1]: mysql.enable flag is not working with v1 but working with v2.

**Pull request merged**